graph/resolver/schema.resolvers.go

```diff
@@ -26,6 +26,14 @@ func (r *mutationResolver) AddDiscussionParticipant(ctx context.Context, discuss
 		if err != nil || !modCheck {
 			return nil, fmt.Errorf("unauthorized")
 		}
+
+		if discussionParticipantInput.IsAnonymous == true {
```

Comment:
Could simplify these to just `if <blah>` or `if !<blah>`

Reply:
it has to be looped in with the `modcheck` since only mods can be non-anon though